src/main/resources/templates/macros/report/steps.vm

```diff
@@ -25,7 +25,19 @@
         #end
 
         #includeDocString($step.getDocString())
-        #includeOutput($step.getOutput())
+
+        #set($combineOutputs = "true")
+        #foreach($output in $step.getOutput())
+          #if(${output.getMessages().size()} > 1)
+            #set($combineOutputs = "false")
+          #end
+        #end
+        #if($combineOutputs == "true")
```

Comment:
There are no explicit data types in Apache Velocity. So I can't use $combineOutputs as boolean variable.